server/public/styles.css

```diff
@@ -281,6 +291,19 @@ img:hover {
 
   .results {
     font-size: 24px;
-    padding-left: 20px;
+    margin: 20px;
+  }
+
+  section {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+  }
+
+  .venue-card {
+    display: flex;
```

Comment:
Please test all of your CSS properties. You have some duplicates in your code and also some that are not needed at all. Try removing them one-by-one to confirm which ones you can delete.